test/functional/ios/testapp/device-specs.js

```diff
@@ -60,23 +60,32 @@ describe('testapp - device', function () {
     });
   });
 
-  _.each(['iPhone', 'iPad'], function (device) {
-    describe('generic ' + device + ' deviceName @skip-ios6', function () {
+  describe('generic deviceName @skip-ios6', function () {
```

Comment:
Ah, there was a failing test around here somewhere and I was suspicious if this contained a weird race condition (or dropped a callback). It wasn't the issue but I liked the more-declarative wetter version better after I was done.

I'd go back to the dry way if there were three tests, but for just two it seemed too complex for just test code.